packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -205,6 +209,55 @@ def generate_vip_app_permutations():
             for proxy_net in list(marathon.Network)]
 
 
+def workload_test(dcos_api_session, container, app_net, proxy_net, ipv6, same_host):
+    (vip, hosts, cmd, origin_app, proxy_app) = \
+        vip_workload_test(dcos_api_session, container,
+                          app_net, proxy_net, ipv6, True, same_host)
+    return (hosts, origin_app, proxy_app)
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize('same_host', [True, False])
+def test_ipv6(dcos_api_session, same_host):
+    ''' Testing autoip, containerip and *.mesos FQDN on ipv6 overlay network '''
+    (hosts, origin_app, proxy_app) = \
+        workload_test(dcos_api_session, marathon.Container.DOCKER,
+                      marathon.Network.USER, marathon.Network.USER, True, same_host)
+    log.info('Starting apps :: Hosts: {}'.format(hosts))
+    log.info("Origin app: {}".format(origin_app))
+    origin_app.deploy(dcos_api_session)
+    log.info("Proxy app: {}".format(proxy_app))
+    proxy_app.deploy(dcos_api_session)
+    origin_app.wait(dcos_api_session)
+    proxy_app.wait(dcos_api_session)
+    log.info('Apps are ready')
+    origin_app_info = origin_app.info(dcos_api_session)
+    origin_port = origin_app_info['app']['container']['portMappings'][0]['containerPort']
+    proxy_host, proxy_port = proxy_app.hostport(dcos_api_session)
+    try:
+        zones = ["marathon.autoip.dcos.thisdcos.directory",
+                 "marathon.containerip.dcos.thisdcos.directory",
+                 "marathon.mesos"]
+        for zone in zones:
+            cmd = '{} --ipv6 http://{}/test_uuid'.format(
+                '/opt/mesosphere/bin/curl -s -f -m 5',
+                '{}.{}:{}'.format(origin_app.id, zone, origin_port))
+            log.info("Remote command: {}".format(cmd))
+            ensure_routable(cmd, proxy_host, proxy_port)['test_uuid'] == origin_app.uuid
+    finally:
+        log.info('Purging application: {}'.format(origin_app.id))
+        origin_app.purge(dcos_api_session)
+        log.info('Purging application: {}'.format(proxy_app.id))
+        origin_app.purge(dcos_api_session)
+
+
+@pytest.mark.slow
+@pytest.mark.skip(reason='DCOS_OSS-1993')
```

Comment:
Hey @urbanserj  - Why is this skipped ?  Just noticing. This is not exercised at all.

Reply:
@orsenthil Linux kernel crashes on this test (see [DCOS_OSS-1993](https://jira.mesosphere.com/browse/DCOS_OSS-1993)), so vips for ipv6 are disabled by default.

Reply:
Thanks, I forgot about that. Also, @urbanserj  - that bug is closed - Please reopen with the proper status.